src/App.js

```diff
@@ -5,14 +5,47 @@ import Header from './components/header/header.component';
 import Homepage from './pages/homepage/homepage.component';
 import MapPage from './pages/map/map.component';
 import AboutPage from './pages/about/about.component';
-
 import './App.css';
 
+// npm install react-particles-js
+import Particles from 'react-particles-js';
+import 'tachyons';
```

Comment:
Already import this in index.js, please remove it from here.

Reply:
uhmm If I take it out from App.js I'm having troubles with everything because I already build considering App.js.
I deleted tachyons from index.js instead

Reply:
Actually you shouldn't be, index.js is one level higher than App.js, but what's done it's done. It also works that way.